code/obj/item/device/pda2/smallprogs.dm

```diff
@@ -808,6 +808,14 @@ Code:
 
 		src.post_signal(signal)
 
+		if(isliving(usr))
+			playsound(src.master, 'sound/items/security_alert.ogg', 60)
+			var/map_text = null
+			map_text = make_chat_maptext(usr, "Emergency alert sent.", "font-family: 'Helvetica'; color: #D30000; font-size: 7px;", alpha = 215)
+			for (var/mob/O in hearers(usr))
+				O.show_message(assoc_maptext = map_text)
+			usr.audible_message("<span class='alert'>[usr] presses a red button on the side of their [src.master].</span>", "<span class='notice'>You press the \"Alert\" button on the side of your [src.master].</span>", "<span class='alert'>You see [usr] press a button on the side of their [src.master].</span>")
```

Comment:
why is *this* an audible message though

Reply:
Since I think Im responsible for this code, I imagined the maptext coming from a robotic voice from the pda itself, a message in chat reinforcing this I think would be good.

Reply:
nah, this one's on me; I replaced visible message with audible message after mistaking what Ze was telling me to revise